src/lite-yt-embed.js

```diff
@@ -40,6 +39,19 @@ class LiteYTEmbed extends HTMLElement {
         // TODO: support dynamically setting the attribute via attributeChangedCallback
     }
 
+    fetchYoutubeAPI(cb) {
+        var tag = document.createElement('script');
+        tag.src = 'https://www.youtube.com/iframe_api';
+        tag.async = true;
+        var firstScriptTag = document.getElementsByTagName('script')[0];
```

Comment:
We shouldn't be manipulating things outside of the element itself. Please just add the script within the element's scope.

Reply:
I don't agree in this case, because you're fetching the library on demand